### PR TITLE
Update tiled_ru.ts

### PR DESCRIPTION
**Pull request by semtiko**
Thank you for your work!

### CODE REVIEW

File: translations/tiled_ru.ts [hunk 1]
```diff
@@ -2357,44 +2357,44 @@ Line %1, column %2</source>
...
         <location line="+11"/>
         <source>Image too Big</source>
-        <translation type="unfinished"></translation>
+        <translation>Изображение слишком большое</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The resulting image would be %1 x %2 pixels and take %3 GB of memory. Tiled is unable to create such an image. Try reducing the zoom level.</source>
-        <translation type="unfinished"></translation>
+        <translation>Конечное изображение имеет размеры %1 x %2 пикселей и требует %3 GB памяти. Tiled не удалось создать изображение. Попробуйте увеличить масштаб.</translation>
```
Comment (bjorn) [inline translations/tiled_ru.ts]: Google translate translates the last part as "Try to zoom in.", but reducing the zoom level means zooming out. Is this translation wrong or Google translate being silly?

**Comment by bjorn** [comment]
Thank you for the update! I checked a few translations in Google Translate and found one case where I think the translation may be wrong, but I have personally no way to judge it so please verify.

**Comment by semtiko** [comment]
Sorry for long reply. In russian language "try to zoom in" it's like a word game and can be translated in few ways and my way is just a simple way, but without missing something important. Don't worry )

**Comment by bjorn** [comment]
Alright, thanks again!

**Pull request merged**